### ISSUE TITLE
Decide what to do with the no-reply email address

### ISSUE THREAD

**Issue by jfly** [thewca/wca-forum#11]
Right now, the forum is using email address `noreply@forum.worldcubeassociation.org` for all emails:

![image](https://user-images.githubusercontent.com/277474/62119674-f05ba880-b274-11e9-914a-e28d85349f7b.png)

I wonder if this email address is configurable? If not, maybe we should set up an email address at `noreply@forum.worldcubeassociation.org` somehow so emails are never lost.

**Comment by jfly** [comment]
If we do decide to keep this email address, I think we'll need to confirm it with Amazon SES, or risk getting limited/put in spam folders/something.

**Comment by jfly** [comment]
Here's what happens right now if you reply to that email address:

![image](https://user-images.githubusercontent.com/277474/62269310-8283be00-b3e7-11e9-9bb1-158513254a7e.png)

**Comment by AlbertoPdRF** [comment]
Yes, these are very annoying :/ I think we should keep this email address, but it's probably not worth it that someone monitors its inbox.
